Microsoft.Toolkit.Uwp.UI.Controls/Carousel/Carousel.cs

```diff
@@ -234,6 +234,11 @@ private void FocusContainerFromIndex(int index)
         }
 
         /// <summary>
+        /// Occurs when the selected item has changed
+        /// </summary>
+        public event EventHandler<object> SelectionChanged = null;
```

Comment:
`SelectionChangedEventHandler` has removed and added items.

    public class SelectionChangedEventArgs : RoutedEventArgs, ISelectionChangedEventArgs
    {
        public SelectionChangedEventArgs(IList<object> removedItems, IList<object> addedItems);

        public IList<object> AddedItems { get; }
        public IList<object> RemovedItems { get; }
    }

Since the carousel can't have multiple items selected at the same time, I don't think that this event could be useful.
The only thing that could be useful is, maybe, the unselected element.
For instance, a classic `EventHandler` seems more easier to use.
Make sense ?